systemtest/src/main/java/io/strimzi/systemtest/templates/crd/KafkaTemplates.java

```diff
@@ -232,18 +226,18 @@ private static KafkaBuilder defaultKafka(Kafka kafka, String name, int kafkaRepl
     public static KafkaBuilder kafkaFromYaml(File yamlFile, String clusterName, int kafkaReplicas, int zookeeperReplicas) {
         Kafka kafka = getKafkaFromYaml(yamlFile);
         return new KafkaBuilder(kafka)
-            .withNewMetadata()
+                .withNewMetadata()
                 .withName(clusterName)
                 .withNamespace(kubeClient().getNamespace())
-            .endMetadata()
-            .editOrNewSpec()
+                .endMetadata()
+                .editOrNewSpec()
                 .editKafka()
-                    .withReplicas(kafkaReplicas)
+                .withReplicas(kafkaReplicas)
                 .endKafka()
                 .editZookeeper()
-                    .withReplicas(zookeeperReplicas)
+                .withReplicas(zookeeperReplicas)
                 .endZookeeper()
-            .endSpec();
+                .endSpec();
     }
```

Comment:
If I'm seeing this correctly, you broke the indentation, can you please revert it back?

Reply:
Yeah, idea did. Fixing everything here.